ui/job-view/App.jsx

```diff
@@ -217,6 +236,32 @@ class App extends React.Component {
     return repo;
   }
 
+  async setLandoRevision() {
+    const { pushRoute } = this.props;
+    const params = getAllUrlParams();
+    const landoCommitID = params.get('landoCommitID');
+    let revision = params.get('revision');
```

Comment:
Line 247 overwrites the value. This line can be removed with the definition moved down.